raiden_contracts/data/source/services/MonitoringService.sol

```diff
@@ -400,7 +400,7 @@ contract MonitoringService is Utils {
                 "\x19Ethereum Signed Message:\n221",  // 20 + 32 + 32 + 20 + 20 + 65 + 32
                 address(this),
                 chain_id,
-                uint256(MessageTypeId.MSReward),
+                uint256(6),
```

Comment:
Not so nice, any better ideas?

Reply:
Does the compiler resolve enums during compilation? If that is the case, you should be able to have the enum in Utils (or a similar superclass if Utils contains too much baggage) and use it everywhere without it increasing the size of the resulting contracts.

Reply:
[The docs](https://docs.soliditylang.org/en/latest/types.html#enums) don't say it explicitly, but it sounds like the enums are resolved to the respective ints during compilation. So it might be possible to add the enum to all contracts without increasing their size.

Reply:
The problem is that libraries are not allowed to inherit:
```
Error: Library is not allowed to inherit.
 --> data/source/lib/TokenNetworkUtils.sol:8:1:
  |
8 | library TokenNetworkUtils is Utils{
  | ^ (Relevant source part starts here and spans across multiple lines).
make: *** [compile_contracts] Error 1

Reply:
But I can try pushing it into another lib like `ECVerify`

Reply:
Ahh, I didn't know that. You could also copy the Enum and add a comment like `when changing this, please also update the copy in otherFile.sol`. Not very elegant, but better than having the plain numbers, IMO.

Reply:
That's true. 

I added another small lib now: https://github.com/raiden-network/raiden-contracts/pull/1437/files#diff-76de3909eb0a0b12081b5e019df894b1f383260355d1f9051efb92ac6dc97929R7

That seems to work fine and is a good solution IMO.